migtests/tests/import-file/run-import-file-test

```diff
@@ -284,9 +288,18 @@ main() {
 
 	step "Run validations."
 	"${TEST_DIR}/validate"
+
+	# End Migration doesn't work in dockerised environment with import data file
```

Comment:
try `import data status` command as well, I would suspect it will also be the case.

Reply:
https://yugabyte.atlassian.net/browse/DB-16274